docs/src/js/namespaces/embedding/classes/EmbeddingFunctionRegistry.md

```diff
@@ -145,3 +167,27 @@ reset the registry to the initial state
 #### Returns
 
 `void`
+
+***
+
+### setVar()
+
+```ts
+setVar(name, value): void
+```
+
+Set a variable. These can be accessed in the embedding function
+configuration using the syntax `$var:variable_name`. If they are not
+set, an error will be thrown letting you know which key is unset. If you
+want to supply a default value, you can add an additional part in the
+configuration like so: `$var:variable_name:default_value`.
```

Comment:
Why would I set a default value in the `setVar` function?

Reply:
It probably doesn't make sense for secrets, but for runtime configuration like `device` I feel like it's nice to provide a fallback value. For the `device` example, I show using `cpu` as a fallback default value.